README.md

```diff
@@ -1 +1,102 @@
+<div>
+    <img alt="state-mate" src="https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fraw.githubusercontent.com%2Flidofinance%2Fstate-mate%2Fmain%2Fpackage.json&query=%24.version&label=state-mate&labelColor=white&color=green"/>
+    <img alt="Node.js" src="https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fraw.githubusercontent.com%2Flidofinance%2Fstate-mate%2Fmain%2Fpackage.json&query=%24.engines.node&style=flat&label=node.js&labelColor=rgb(62%2C%20109%2C%2026)&color=white"/>
+    <img alt="GitHub license" src="https://img.shields.io/github/license/lidofinance/state-mate?labelColor=orange&color=white"/>
+</div>
+
 # state-mate
+
+state-mate is a simple tool that validates contracts' states against a concise YAML-based description. Run state-mate to verify deploy outcome, current state, access control and more.
+
+state-mate accepts a yaml file that includes contract addresses, view functions and their expected results. It calls each function and compares the output to the expected result.
+
+## ✨ Features
+
+- state (non-mutable functions result) verification,
+- automatically validates that all functions covered,
+- supports any EVM network,
+- easily configurable checks,
+- CI-friendly,
+- uses yaml config (env variables, variables, comments).
+
+## ⚡ Getting started
+
+### Requirements
+
+- git
+- Node.js >=20,
+- yarn 1.x
```

Comment:
Wow, shall we use modern yarn instead, given the up-to-date LTS node.js is required?

Reply:
Technically any yarn will work, so I removed the version